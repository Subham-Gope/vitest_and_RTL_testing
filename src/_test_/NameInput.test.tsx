// import { describe, test, expect } from "vitest";
// import { render, screen } from "@testing-library/react";
// import NameInput from "../components/NameInput";

// describe("NameInput.tsx", () => {
//   test("testing for the presence of a input filed with placeholder text of 'Enter name'", () => {
//     const { container } = render(<NameInput />);

//     expect(container).toMatchSnapshot();

//     const inputEl = screen.getByPlaceholderText("Enter Name");

//     expect(inputEl).toBeInTheDocument();
//   });
// });
