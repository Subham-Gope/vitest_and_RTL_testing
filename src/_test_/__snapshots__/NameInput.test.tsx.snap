// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`NameInput.tsx > testing for the presence of a input filed with placeholder text of 'Enter name' 1`] = `
<div>
  <input
    placeholder="Enter Name"
    type="text"
  />
</div>
`;
